site/en/install/ide.md

```diff
@@ -96,8 +96,11 @@ Eclipse projects.
 
 ### C Language Family (C++, C, Objective-C, and Objective-C++)
 
-[`hedronvision/bazel-compile-commands-extractor`](https://github.com/hedronvision/bazel-compile-commands-extractor) enables autocomplete in a wide variety of editors. It lets language servers, like clangd and other tooling, draw upon Bazel's understanding of how `cc` and `objc` code will be compiled, including how it configures cross-compilation for other platforms.
+[`hedronvision/bazel-compile-commands-extractor`](https://github.com/hedronvision/bazel-compile-commands-extractor) enables autocomplete in a wide variety of extensible editors, including VSCode, Vim, Emacs, Sublime, etc. It lets language servers, like clangd, ccls, and other tooling, draw upon Bazel's understanding of how `cc` and `objc` code will be compiled, including how it configures cross-compilation for other platforms.
```

Comment:
Content-wise this looks fine to me. 

From a grammar nitpcking standpoint, consider tightening the phrasing like:

- "including VSCode, Vim, Emacs, and Submline" ("etc." is redundant with the word "including").

By "other tooling" do you mean non-language server tooling?  If so, perhaps "It lets language servers like clangd and ccls, and other tooling, ..."? That phrasing still doesn't roll off the tongue but I'm trying to support comma reduction when possible.

Reply:
Great edits all, Greg. Thanks :) Done.
[Yes, indeed: other *types* of tooling. Wanted to get that in there because the compilation databases are a general, build-system-independent nexus for all kinda of tooling, but I also figured I shouldn't get people into the details from the IDE doc. clang-tidy and SonarQube are some examples of other things people are using the tool for.]